### PR TITLE
Try to solve 4.14 build don't pass problem 

### PR DESCRIPTION
**Pull request by chen2021-web**

### CODE REVIEW

File: 核心/ksud.c [hunk 1]
```diff
@@ -167,6 +167,8 @@ int ksu_handle_execveat_ksud(int *fd, struct filename **filename_ptr,
 				char first_arg[16];
 				#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)
 				strncpy_from_user_nofault(first_arg, p, sizeof(first_arg));
+                                #elif (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 14, 0) && LINUX_VERSION_CODE <= KERNEL_VERSION(5, 8, 0))
+				strncpy_from_user(first_arg, p, sizeof(first_arg));
 				#else
 				strncpy_from_unsafe_user(first_arg, p, sizeof(first_arg));
```
Comment (tiann) [inline 核心/ksud.c:173]: I think change `strncpy_from_unsafe_user` to `strncpy_from_user` is ok.

**Review by tiann — CHANGES_REQUESTED** [review]
.

**Comment by tiann** [comment]
```
#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 8, 0)
strncpy_from_user_nofault(first_arg, p, sizeof(first_arg));
#elif LINUX_VERSION_CODE >= KERNEL_VERSION(5, 3, 0)
strncpy_from_unsafe_user(first_arg, p, sizeof(first_arg));
#else
strncpy_from_user(first_arg, p, sizeof(first_arg));
#endif
```

**Review by tiann — APPROVED** [review]

File: 核心/ksud.c (hunk 1 above)
Comment (chen2021-web) [reply 核心/ksud.c]: Should

**Pull request merged**